beebop/resources/args.json

```diff
@@ -45,6 +45,7 @@
             "fulldb": "GPS_v9",
             "external_cluster_prefix": "GPSC",
             "external_clusters_file": "GPS_v9_external_clusters.csv",
+            "db_metadata_file": "GPS_metadata.csv",
```

Comment:
I wonder if this filename should be tagged to the db version..? Or is the idea that this will contain metadata for all samples in all versions of the db?

Reply:
yeah its not really version-specific